These files are private to quilt. I'm not even sure why intltool-extract is
seeing them.
---
 po/POTFILES.skip |   17 +++++++++++++++++
 1 file changed, 17 insertions(+)

Index: gthumb-3.2.3/po/POTFILES.skip
===================================================================
--- gthumb-3.2.3.orig/po/POTFILES.skip	2013-08-31 06:29:10.315631466 +1000
+++ gthumb-3.2.3/po/POTFILES.skip	2013-08-31 06:29:10.307631426 +1000
@@ -1,3 +1,20 @@
 data/gthumb.desktop.in
 data/gthumb-import.desktop.in
 extensions/webalbums/albumtheme.c
+
+# These are the original files patched by quilt.
+.pc/04-fix_source_typo.patch/extensions/edit_metadata/callbacks.c
+#.pc/07-fix_zooms.patch/src/gth-window-actions-callbacks.c
+#.pc/07-fix_zooms.patch/src/gth-window-actions-entries.h
+#.pc/08-fix_preferences.patch/data/glade/gthumb_preferences.glade
+#.pc/09-fix_infinite_loop_dragging_parent_dir.patch/src/gth-browser.c
+#.pc/11-fix_Film_on_import.patch/src/dlg-photo-importer.c
+#.pc/12-remove_Save_button_in_crop_dialog.patch/data/glade/gthumb_crop.glade
+#.pc/12-remove_Save_button_in_crop_dialog.patch/src/dlg-crop.c
+.pc/13-fix_Close_mismatch.patch/gthumb/gth-window-actions-entries.h
+#.pc/14-add_rotate_button_viewer.patch/src/gth-browser.c
+#.pc/15-suggest_new_name_on_overwrite_dialog.patch/src/dlg-file-utils.c
+#.pc/16-fix_crop_dialog_usability.patch/data/glade/gthumb_crop.glade
+#.pc/16-fix_crop_dialog_usability.patch/src/dlg-crop.c
+#.pc/17-dont_follow_symlinks_on_dups_search.patch/src/dlg-duplicates.c
+
